patch/kconfig-inclusions

```diff
@@ -162,6 +162,8 @@ CONFIG_MTD_RAW_NAND=m
 CONFIG_MTD_NAND_ECC_SW_BCH=y
 CONFIG_MTD_NAND_MARVELL=m
 CONFIG_SENSORS_EMC2305=m
+CONFIG_PHY_MVEBU_CP110_COMPHY=y
```

Comment:
Any reason this can't be compiled as a module instead of being included in the kernel binary itself?

Reply:
@saiarcot895 As you might know by default this config is compiled as module. We saw a kernel hang while booting this on boards using CP110 comphy. Below are short snippet of logs. Hardware probe issue is resolved by compiling with 'y'.

[    2.224925] armada38x-rtc f2284000.rtc: registered as rtc0
[    2.234564] armada8k-pcie f2600000.pcie: host bridge /cp0/pcie@f2600000 ranges:
[    2.242347] mv64xxx_i2c f2701000.i2c: mv64xxx: missing ARB-lost recovery defs in dts file
[    2.243754] SCSI subsystem initialized
[    2.244543] usbcore: registered new interface driver usbfs
[    2.244558] usbcore: registered new interface driver hub
[    2.244585] usbcore: registered new device driver usb
[    2.245708] armada8k-pcie f2600000.pcie:      MEM 0x00c0000000..0x00dfefffff -> 0x00c0000000
[    2.245785] armada8k-pcie f2600000.pcie: Link up
[    2.245869] armada8k-pcie f2600000.pcie: PCI host bridge to bus 0000:00
[    2.245872] pci_bus 0000:00: root bus resource [bus 00-ff]
[    2.245875] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xdfefffff]
[    2.245894] pci 0000:00:00.0: [11ab:0110] type 01 class 0x060400
[    2.245902] pci 0000:00:00.0: reg 0x10: [mem 0x00000000-0x000fffff]
[    2.245948] pci 0000:00:00.0: supports D1 D2
[    2.245950] pci 0000:00:00.0: PME# supported from D0 D1 D3hot
<Hangs here>

Reply:
In that case, it might be worthwhile to remove the tristate option for this config flag in upstream, if it needs to be in the kernel binary.